Makefile

```diff
@@ -22,7 +22,7 @@ ifeq ($(os_distro), al2023)
 	AMI_VARIANT := $(AMI_VARIANT)-al2023
 endif
 ifeq ($(arch), arm64)
-	instance_type ?= m6g.large
+	instance_type = m6g.large
```

Comment:
What's the reason for this change?

Reply:
`instance_type` will only been updated if it is not already set when using `?=`

thus run into the following error initially 
```
 make 1.28 os_distro=al2023 arch=arm64         

PACKER_ARGS: -var-file templates/al2023/variables-default.json  **-var instance_type='m5.large'** -var arch='arm64'  -var kubernetes_version='1.28.5'
```

Reply:
oh but we want to avoid locking in `instance_type`, any conflicts with this?
```Make
ifeq ($(arch), arm64)
	instance_type ?= m6g.large
	AMI_VARIANT := $(AMI_VARIANT)-arm64
else
	instance_type ?= m5.large
endif
```

Reply:
ah, it would be better, although it has been defaulted to m5.large at line 19 already 😄
will update it in next rev shortly